data/docs/api.mdx

```diff
@@ -83,7 +83,7 @@ A function to generate styles on the [server-side](/docs/server-side-rendering).
 
 ## defaultThemeMap
 
-You can import `defaultThemeMap` if you want to merge with the [default mapping](https://github.com/modulz/stitches/blob/canary/packages/core/src/defaultThemeMap.js#L11-L155).
+You can import `defaultThemeMap` if you want to merge with the [default mapping](https://github.com/modulz/stitches/blob/canary/packages/core/src/default/defaultThemeMap.js#L16-L162).
```

Comment:
hey @peduarte, yeah if this is gonna change then probably better to drop them, here's a suggestion:

```suggestion
You can import `defaultThemeMap` if you want to merge with the [default mapping](https://github.com/modulz/stitches/blob/canary/packages/core/src/default/defaultThemeMap.js).
```